app/views/admin/sidebar/_target_sidebar.html.erb

```diff
@@ -1,4 +1,4 @@
-<li id="sidebar_<%= sidebar.id %>" class="sidebar_item" data-sortable="sidebar_<%= sidebar.id %>" >
+<li id="sidebar_<%= sidebar.id %>" class="sidebar-item" data-sortable="sidebar_<%= sidebar.id %>" >
```

Comment:
Will not merge.

This fix introduces new bugs in existing styles and javascript.

See:
app/assets/stylesheets/sidebar_admin.css.scss#L25
app/assets/javascripts/sidebar.js#L31